### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by JKramer91**

### CODE REVIEW

**Review by lanaitu — APPROVED** [review]
Good description of Chirp!

File: README.md [hunk 1]
```diff
@@ -1,5 +1,62 @@
...
...ccount can be created in one of two ways. Either an account can be created by simply choosing a username, email and password. Alternatively, an account can be created through the use of GitHub authorization, thus using the credentials of ones GitHub account. After an account is created, the new user is able to fully interact with the application. The user can send a new cheep by inputting text in the input form above the timeline and then clicking on the 'Share' button. Additionally, the user can choose to follow other users and/or like their cheeps by clicking the 'Follow' and 'Like' buttons respectively. In the header of the application, additional buttons are located giving the user freedom to navigate the site. The 'Home' button simply redirects to the homepage, thereby showing the public timeline. The '[username]'s Timeline' button redirects to a users private timeline. The 'Your Cheeps' button shows the cheeps authored by the user currently existing in the application. The 'About me' button shows the information, for example the name and email, that the application currently stores about the user. Finally, by clicking the 'Logout' button, the user logs out of the application.
```
Comment (ClaraWJ) [inline README.md:28]: he or she --> 'they are'

File: src/Chirp.Web/Program.cs
Comment (ClaraWJ) [inline src/Chirp.Web/Program.cs:1]: Skal kommenteres ud igen.

**Review by ClaraWJ — CHANGES_REQUESTED** [review]
she or he --> they are
EnsureDeleted skal kommenteres ud igen.

**Review by ClaraWJ — APPROVED** [review]
Great readme, well done!

**Pull request merged**